includes/class-wc-gateway-swedbank-pay-invoice.php

```diff
@@ -192,11 +219,10 @@ public function payment_fields() {
 		?>
 		<p class="form-row form-row-wide">
 			<label for="social-security-number">
-				<?php echo __( 'Social Security Number', WC_Swedbank_Pay::TEXT_DOMAIN ); ?>
+				<?php echo __( 'Social Security Number', 'swedbank-pay-woocommerce-payments' ); ?>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '__'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)